doc/getting_started/installation_linux.rst

```diff
@@ -64,7 +64,7 @@ Install the required packages in a Fedora host system with:
    $ sudo dnf group install "Development Tools"
    $ sudo dnf install git make gcc glibc-static \
 	 libstdc++-static python3-ply ncurses-devel \
```

Comment:
Anas added it in some other patch.  I noticed this when i rebased.  On fedora it is dtc